src/services/config.js

```diff
@@ -142,6 +143,18 @@ class ConfigData {
     prompt.acked = true;
   }
 
+  setPluginData(pluginName, data) {
```

Comment:
Overall, I like this idea and the clean naming convention. 

With this setup, it seems like we need to be careful with name spacing to avoid unintended data manipulation. Perhaps the `pluginName` needs to be `<org>/<name of plugin>`.

Reply:
There's not much we can do about data manipulation, any plugin can overwrite the entire `userConfig` if it wanted to. The idea that setting the plugin name (which could be defined as "serverless" or "@twilio-labs/plugin-serverless", for example) should be a unique namespace.

A better alternative would be to have the function `setPluginData(data)` and the name get read from the plugin itself. But I don't know if plugin name/namespace is available to the plugin during runtime. Do you know if that is possible?